src/components/player-area/player-areas-utils.js

```diff
@@ -23,6 +24,11 @@ function calculateVerticalDimensions(options) {
 
   let videoWidth = playerWidth - verticalPanelCount * verticalPanelWidth;
 
+  if (playerWidth < PLAYER_BREAK_POINTS.SMALL) {
```

Comment:
why are you making the calculation again if you can get PLAYER_SIZE from state.shell ?